x-pack/plugins/security_solution/public/timelines/components/edit_data_provider/helpers.tsx

```diff
@@ -8,9 +8,10 @@
 import { findIndex } from 'lodash/fp';
 
 import type { EuiComboBoxOptionOption } from '@elastic/eui';
+import type { FieldCategory } from '@kbn/timelines-plugin/common/search_strategy';
 import { DataProviderType } from '../../../../common/api/timeline';
 
-import type { BrowserField, BrowserFields } from '../../../common/containers/source';
+import type { BrowserFields } from '../../../common/containers/source';
```

Comment:
absolutely not important, but if you're making more changes to this PR, you can remove the empty lines above and below